projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/base-field/grey-bar.scss

```diff
@@ -1,5 +1,7 @@
 .form :host::ng-deep .grey-bar {
-  > * > .form-group, > * > dl.case-field {
+  > * > .form-group,
```

Comment:
This probably works, but seems fiddly.
I'm not a CSS expert, but isn't this just trying to say "dl.case-field as a descendent of grey-bar". 
so you don't seem to need all the malarky with combinators - wouldn't 
.grey-bar .dl.case-field
or possibly
.grey-bar .dl.case-field,
.grey-bar .form-form-group .dl.case-field

work?

Reply:
Andy is right - ng-deep is deprecated. Try looking at :host-context:
https://blog.angular-university.io/angular-host-context/

Reply:
I haven't touched any of that and kept changes to a minimal, I've only added `> * > * > .form-group`